proto/api/v2/report_service.proto

```diff
@@ -57,6 +57,7 @@ message VulnerabilityReportFilters {
     google.protobuf.Timestamp since_start_date = 6;
   }
   bool include_nvd_cvss = 7;
+  bool includeEpssProbability= 8;
```

Comment:
Should probably name it in sanke case, similar to the nvd cvss one

Reply:
Good point. I have trained myself to do that conversion from camelCase to Find in Files source of truth.

Also looks like a missing space preceding `=`